### PR TITLE
replicaset fix typo

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
reopen #47821
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
@NickrenREN 
**Release note**:

```release-note
   NONE
```

### CODE REVIEW

File: pkg/controller/replicaset/replica_set_utils.go [hunk 1]
```diff
@@ -126,7 +126,7 @@ func calculateStatus(rs *extensions.ReplicaSet, filteredPods []*v1.Pod, manageRe
 	return newStatus
 }
 
-// NewReplicaSetCondition creates a new replica set condition.
+// NewReplicaSetCondition creates a new replicaset condition.
```
Comment (dixudx) [inline pkg/controller/replicaset/replica_set_utils.go]: Well, I think it's okay to use `replica set` in comments, not typo issue.

Comment (jianglingxia) [inline pkg/controller/replicaset/replica_set_utils.go]: but the kind: ReplicaSet ，I look the doc in https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
both use ReplicaSet objects

**Comment by jianglingxia** [comment]
ping @kargakis  @caesarxuchao

File: pkg/controller/replicaset/replica_set_utils.go (hunk 1 above)
Comment (caesarxuchao) [inline pkg/controller/replicaset/replica_set_utils.go]: Beauty is in the eye of the beholder. I like "replicaset" as one word better, so i'm going to lgtm this.

**Pull request merged**